wandb/sdk/wandb_run.py

```diff
@@ -3581,66 +3563,76 @@ def link_model(
-            located at 'path' or log a new version. Otherwise log files under 'path' as a new model artifact, 'name'
-            of type 'model'.
-            - Check if registered model with name 'registered_model_name' exists in the 'model-registry' project.
-            If not, create a new registered model with name 'registered_model_name'.
-            - Link version of model artifact 'name' to registered model, 'registered_model_name'.
-            - Attach aliases from 'aliases' list to the newly linked model artifact version.
+        First, check if `name` model artifact has been logged. If so,
+        use the artifact version that matches the files
+        located at `path` or log a new version. Otherwise log files
+        under `path` as a new model artifact, `name`
+        of type "model".
 
-        Args:
-            path: (str) A path to the contents of this model,
-                can be in the following forms:
-                    - `/local/directory`
-                    - `/local/directory/file.txt`
-                    - `s3://bucket/path`
-            registered_model_name: (str) - the name of the registered model that the model is to be linked to.
-                A registered model is a collection of model versions linked to the model registry, typically representing a
-                team's specific ML Task. The entity that this registered model belongs to will be derived from the run
-            name: (str, optional) - the name of the model artifact that files in 'path' will be logged to. This will
-                default to the basename of the path prepended with the current run id  if not specified.
-            aliases: (List[str], optional) - alias(es) that will only be applied on this linked artifact
-                inside the registered model.
-                The alias "latest" will always be applied to the latest version of an artifact that is linked.
-
-        Examples:
-            ```python
-            run.link_model(
-                path="/local/directory",
-                registered_model_name="my_reg_model",
-                name="my_model_artifact",
-                aliases=["production"],
-            )
-            ```
+        Next, check if registered model with name `registered_model_name`
+        exists in the 'model-registry' project.
 
-            Invalid usage
-            ```python
-            run.link_model(
-                path="/local/directory",
-                registered_model_name="my_entity/my_project/my_reg_model",
-                name="my_model_artifact",
-                aliases=["production"],
-            )
+        If not, create a new registered model with name `registered_model_name`.
+        - Link version of model artifact `name` to registered model, `registered_model_name`.
+        - Attach aliases from 'aliases' list to the newly linked model artifact version.
```

Comment:
This reads like instructions to the user because of the imperative mood.

It's kind of strange for this to appear in the docstring since it's describing the implementation. It should at most be a comment in the code, right below the docstring, but it looks outdated anyway, so maybe remove?